projects/observability/src/shared/components/span-detail/call/headers/span-detail-call-headers.component.scss

```diff
@@ -1,9 +1,12 @@
 @import 'font';
 
+:host {
```

Comment:
can we avoid doing host styling?

Reply:
We'll have to add this is parent which I don't think is better than adding this host styling here.

Reply:
@adisreyaj why not? why is `display: block`  needed here?